InscryptionAPI/Triggers/Interfaces.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using DiskCardGame;
+
+namespace InscryptionAPI.Triggers;
+
+public interface IAddedToHand
```

Comment:
I assume it was to prevent IEnumerator allocations, since otherwise an enumerator would be allocated for every trigger on every card even if they do nothing.

The interface replaces the need for that, if you don't need a certain trigger, you just don't add the interface for it

Reply:
The established pattern also allows you to conditionally respond to triggers (for example, only respond to the player's upkeep, not the opponent's).

Of course, you can just yield break if you don't want to respond to a specific case, and it's not like these triggers are firing every frame.*

Reply:
Some people wanted to keep it, and its not much code, so I added them back